src/main/java/adris/altoclef/util/KillAura.java

```diff
@@ -67,8 +90,9 @@ public void setRange(double range) {
     private boolean attack(AltoClef mod, Entity entity) {
         if (entity == null) return false;
         if (Double.isInfinite(_forceFieldRange) || entity.squaredDistanceTo(mod.getPlayer()) < _forceFieldRange * _forceFieldRange) {
+            mod.getInventoryTracker().swapToWeapon();
```

Comment:
This will be applied to ALL attacks, including the FASTEST mode. In that case, the weapon's durability will deplete extremely quickly, so we don't want this. Consider passing a boolean into this function that tells it whether to use a sword or not

Reply:
now it checks if the strategy is DELAY